### PR TITLE
Add extra documentation for app.import

### PR DESCRIPTION
**Pull request by matteddy**
I struggled for a while to figure out how to import a web worker and eventually stumbled across [this merged RFC](https://github.com/ember-cli/rfcs/pull/28/files). 

It is very possible this is documented elsewhere and I missed it, but this seems like the appropriate place for the information. 😅

### CODE REVIEW

**Comment by mansona** [comment]
Hey @matteddy 👋  thanks for the contribution.

While I'm not against adding documentation for something that actually works with ember-cli, I'm pretty sure that the "recommended" way to do things is to use ember-auto-import these days 🤔  Did you give it a go? Did that also not work for you? 

If ember-auto-import does work for you I wonder if you think we should add something like https://guides.emberjs.com/release/addons-and-dependencies/#toc_regular-npm-packages to the cli-guides as well? 🤔

**Comment by jenweber** [comment]
I think ember-auto-import only helps you out if your file is in an npm package. For serving up individual files in vendor, this is still useful. I vaguely recall that web workers are really particular about how they get imported. 

Even if auto-import helped here, the "advanced" section of the CLI guides is where things like this belong, I think. That way, if someone sees it in an app file they didn't write, they can learn something about it. @mansona is it ok with you to merge?

**Review by jenweber — APPROVED** [review]

File: guides/advanced-use/asset-compilation.md [hunk 1]
```diff
@@ -396,6 +396,18 @@ module.exports = function(defaults) {
 };
 ```
 
+If you would like to specify the output path for a single vendored asset, you can do so by using `app.import`.
```
Comment (jenweber) [inline guides/advanced-use/asset-compilation.md:399]:
```suggestion
If you would like to specify the output path for a single file in the `vendor` directory, you can do so by using `app.import`.
```

This edit should make the spellchecker test pass.

**Review by locks — APPROVED** [review]
Approved by Learning Team committee.

**Comment by amyrlam** [comment]
Related conversation for posterity: https://discord.com/channels/480462759797063690/486548111221719040/842424159610077214

**Pull request merged**